src/utils/api.js

```diff
@@ -31,6 +42,103 @@ export const formatActions = (actionsByDayObject) => {
   });
 };
 
+// Takes array of action objects and returns an object with {key: value} pairs of {hourOfDay: #-of-actions}
+export const getActionsByTimeOfDay = (actions) => {
+  const actionsByTimeOfDay = {
+    [LATE_NIGHT]: 0,
+    [EARLY_MORNING]: 0,
+    [MORNING]: 0,
+    [AFTERNOON]: 0,
+    [EVENING]: 0,
+    [NIGHT]: 0,
+  };
+  actions.forEach((action) => {
+    const actionHourOfDay = action.createdAt.slice(11, 13);
```

Comment:
I would add a comment here to explain what you're slicing b/c it will break if you change the time format.